kythe/go/extractors/config/runextractor/backup/backup.go

```diff
@@ -36,10 +36,13 @@ import (
 	"path/filepath"
 )
 
+// A File is a simple holder for a backup of some file that supports Recovery
+// to restore the original version.
 type File struct {
 	orig, tmp string // file paths
 }
 
+// New backs a given file up to an adjacent temporary file.
```

Comment:
I'd suggest more detail here:

```
// New creates a backup copy of the specified file, located in the same directory.
// The caller should ensure the Release method is called when the backup is no
// longer needed, to clean up.
```

or words to that effect.

Reply:
done